app.geteduroam.geteduroam.yml

```diff
@@ -0,0 +1,110 @@
+id: app.geteduroam.geteduroam
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: geteduroam-gui
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --system-talk-name=org.freedesktop.NetworkManager
+
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: geteduroam
+    buildsystem: simple
+    build-commands:
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/applications/app.geteduroam.Linux.desktop $FLATPAK_DEST/share/applications/app.geteduroam.geteduroam.desktop
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/48x48/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/48x48/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/128x128/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/256x256/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/512x512/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/512x512/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 metainfo/app.geteduroam.geteduroam.metainfo.xml $FLATPAK_DEST/share/metainfo/app.geteduroam.geteduroam.metainfo.xml
```

Comment:
upstream now has the desktop files in app.geteduroam.geteduroam.desktop. same for the icon files (app.geteduroam.geteduroam.png)